### PR TITLE
[MIG][15.0] web_notify

### PR DESCRIPTION
**Pull request by em230418**
Porting notes:
1. Removed notify_default, since it does not pass validation in [1]
2. Modifications to notificaation widgets are removed in prefer of original widget implementation
3. Odoo improved notification usage in [2], so no need to add extra channels that depend on user's id
    
[1] https://github.com/odoo/odoo/blob/21be7ca0c9eef18184d06755ff83337be1e4752e/addons/web/static/src/core/notifications/notification.js#L49
[2] https://github.com/odoo/odoo/commit/543af27c7d6836ffac9e80ff8490b6ddbd849221

### CODE REVIEW

**Review by JrAdhoc — COMMENTED** [review]
Please fix codecov tests for reviewing. Usage test approved.

**Comment by em230418** [comment]
Checked why codecov did not cover tests. https://github.com/OCA/web/pull/2176/checks?check_run_id=5770875580
There was error while executing tests (https://github.com/OCA/web/runs/5770842937?check_suite_focus=true), but for some reason it is marked as passed.

**Comment by matiasperalta1** [comment]
Is this PR ready for review?

**Comment by em230418** [comment]
You can review functionality. Unit-tests are not ready.

File: web_notify/models/res_users.py [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+    def notify_info(self, message="Default message", title=None, sticky=False):
+        title = title or _("Information")
+        self._notify_channel(INFO, message, title, sticky)
+
+    def _notify_channel(
+        self, type_message, message=DEFAULT_MESSAGE, title=None, sticky=False
+    ):
+        # pylint: disable=protected-access
+        if not self.env.user._is_admin() and any(
```
Comment (JosDeGraeve) [inline web_notify/models/res_users.py:38]: This allows to use sudo() to send a message to any user.

```suggestion
        if not (self.env.su or self.env.user._is_admin()) and any(
```

**Review by JosDeGraeve — COMMENTED** [review]
Just a small remark,

Probably also an issue with v14 - i was expecting that using sudo() allows to send a notification to any user.

**Review by matiasperalta1 — APPROVED** [review]

**Comment by em230418** [comment]
Closing in favour of https://github.com/OCA/web/pull/2381

**Pull request closed without merging**